### DISCUSSION TITLE
One way synchronization for selective tables

### DISCUSSION THREAD

**Discussion by truthz03** [Mimetis/Dotmim.Sync#1272] [Q&A]
Hi,

I have a big database with many tables and some of them contains files.
Syncing the file tables into my sqlite webassembly client (browser) is not a good idea because of the memory limitations.

Is it possible to add the file tables to the sync process but only allow to send new files to the server?
After the sync process the table inside the client should be cleared again.

My only alternative for now is to save the files somewhere else in the browser and send them to the server after a synchronization.
But this is a little bit complicate and not really error resistant.

**Comment by Mimetis** [comment]
Make a scope for bidirectional sync (for all tables without files)
And make a scope for the huge table with files, that will be only Upload from client to server.

**Reply by truthz03** [reply]
I tried this now but the problem is that the files are too big. I can not load them into the browser heap space.
I need something to send the files as stream to the backend.

Maybe it is possible to do something linke this:
1.) Subscribe to an Dotmim event inside the client triggering when the sync starts
2.) Send the files as stream to the sync controller where the will be stored temporary
3.) Subscribe to an Dotmim event inside the server triggering when the transaction has opened
4.) Add the tempoaray files to the database after the transation has opened
5.) Finish

**Reply by Mimetis** [reply]
I don't know, but you can try with the interceptors ("`On....()`") maybe, but not sure it will work.
If you find a solution, feel free to share it !
